2023-03-10-3mdeb-msi-pro-z690-a-ddr4-first-qubes-certified-desktop.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "The 3mdeb MSI PRO Z690-A DDR4 is the first Qubes-certified desktop computer!"
```

Comment:
> Please let me know what do you think about that.

It is ungrammatical in English, but I can fix that later. We can just focus on the intended meaning for now. :)

> To optimize titles we use: https://www.aminstitute.com/headline/
It is ok if you don't like that approach, just let me know and will try to create something more similar to what you proposed.

I think a more factual and less emotional tone is more appropriate for Qubes News posts, and I'd like to keep the style consistent for our news feed, but there is some room for discretion within these parameters.

> I think we cannot join 3mdeb and MSI together, because we want to avoid situation in which MSI will say we use their brand for our promotion.
> 
> Very safe approach is to say:
> Dasharo compatible with MSI ....
> MSI with Dasharo ...
> etc.

What is the official name of the product you're selling, though? I think we should just use that official name here (at least in the body text, if not in the title).

Reply:
> I think a more factual and less emotional tone is more appropriate for Qubes News posts, and I'd like to keep the style consistent for our news feed, but there is some room for discretion within these parameters.

Ok. Feel free to propose correct title.

> What is the official name of the product you're selling, though? I think we should just use that official name here (at least in the body text, if not in the title).

This is very good question, but answer is not that straight forward. IMHO we don't sell our version of hardware as other certified vendors. We sell fill PC build based on MSI PRO Z690-A DDR4 mainboard with Dasharo open-source firmware distribution pre-installed. There is no special name for the product and maybe that is our mistake.

As you linked shop link you can see name of position in shop: `MSI PRO Z690-A DDR4 with Dasharo Supporters Entrance – Full PC Build` - what means desktop PC build with MSI mainboard and Dasharo installed. Dasharo Supporters Entrance give ability to get future updates which will be available only for subscribers - others will have to compile from source. We have similar philosophy to Donald Knuth about TeX, that only firmware that get through strict validation we doing in our labs can be called Dasharo, but anyone can take our source code and build firmare themselves. Of course that custom builds should not be called Dasharo since those were not released by 3mdeb nor tested accordingly.

Please let me know if you need more context. We are open-source firmware developers not hardware distributors. Our goal is to improve trustworthiness of all computing devices not becoming hardware vendor.

Reply:
> This is very good question, but answer is not that straight forward. IMHO we don't sell our version of hardware as other certified vendors. We sell fill PC build based on MSI PRO Z690-A DDR4 mainboard with Dasharo open-source firmware distribution pre-installed. There is no special name for the product and maybe that is our mistake.

I think it would be good for the product to have a name, if only so that people have an easy way to refer to it.

> As you linked shop link you can see name of position in shop: `MSI PRO Z690-A DDR4 with Dasharo Supporters Entrance – Full PC Build`

I can use this as the product name, if you like, though it is a bit long and unconventional.

> Please let me know if you need more context. We are open-source firmware developers not hardware distributors. Our goal is to improve trustworthiness of all computing devices not becoming hardware vendor.

I see. That makes sense, but then it makes the status of this as a Qubes-certified computer somewhat unclear. @marmarek, any thoughts?

Reply:
@andrewdavidwong please use following link to shop and all update information behind it: https://3mdeb.com/shop/open-source-hardware/dasharo-fidelisguard-z690-qubes-os-certified/